content/donate/options/index.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "Donate"
+template = "donate-options.html"
+[extra]
+header_message = "Donate"
++++
+
+## Should I Donate Through Every.org or Stripe?
```

Comment:
Missing FAQ: one-time donations.

Reply:
There is already an entry here:

> ## Can I make a one-time donation instead of monthly?
> Yup! Every.org supports this. Click the "custom" link in the table above and select "one time".

But I guess its also worth including in the "which platform should I use" FAQ